test/support/schemas/prefix_schema.exs

```diff
@@ -3,12 +3,12 @@ defmodule PrefixSchema do
   use Absinthe.Schema
 
   query do
-    field :foo do
+    field :foo, :integer do
```

Comment:
I ran into this _exact_ thing earlier today; silent failure to add the field. Definitely need something.